changes/2020-07-15_specify-behavior-for-raw-aes-keyring-for-invalid-EC/change.md

```diff
@@ -0,0 +1,99 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Specify behavior for RawAESKeyring for invalid input EC
```

Comment:
np: I would write this as Raw AES Keyring instead of RawAESKeyring (since the naming conventions might differ language to language). I would probably also replace all uses of `raw-aes-keyring` with `Raw AES keyring`

I would also fully write out EC, since this is its first occurence.